examples/GroveButton/GroveButton.ino

```diff
@@ -19,21 +16,36 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 
 //A 512 offset is required for sub-platforms.  516 corresponds to digital pin 4, or D4.
 #define BUTTON_PIN 516
+void setup();
+void loop();
 
 void setup() {
     //add the Grove Pi+ sub-platform
 	mraa_add_subplatform(MRAA_GROVEPI, "0");
 	// initialize digital pin BUTTON_PIN as an input.
 	pinMode(BUTTON_PIN, INPUT);
 	// DebugSerial should be used via cloud (when you don't have the micro USB cable connected)
-	DebugSerial.begin(9600);
+	#ifdef IS_SERIAL_OR_SSH
```

Comment:
Where is the definition of IS_SERIAL_OR_SSH?
#define IS_SERIAL_OR_SSH TRUE or FALSE should be set at the beginning of the sample for this to have any effect.